Assets/Scripts/UI/HostLobbyButton.cs

```diff
@@ -0,0 +1,15 @@
+using Lobby;
+using UnityEngine;
+
+namespace UI
+{
+    public class HostLobbyButton : MonoBehaviour, IButtonPressHandler
+    {
+        [SerializeField] private LobbyManager LobbyManager;
```

Comment:
If we are only going to have one LobbyManager, it could probably just be a singleton.

Reply:
@JudeWhite I'm not the biggest fan of the Singleton pattern. I like to avoid it unless I can't come up with a better solution. There are some good reasons to avoid using the Singleton pattern in [this](https://stackoverflow.com/questions/137975/what-are-drawbacks-or-disadvantages-of-singleton-pattern) StackOverflow thread.

Reply:
I had a suspicion this might be the case. They've been used quite heavily in some of the places I've worked at before, so I suppose it's a bad habit I ought to start breaking.